common/persistence/tests/cassandra_test.go

```diff
@@ -343,9 +339,11 @@ func testCassandraQueueV2(t *testing.T, cluster *cassandra.TestCluster) {
 		testCassandraQueueV2ConcurrentConflicts(t, cluster)
 	})
 	t.Run("MultiplePartitions", func(t *testing.T) {
-		t.Parallel()
 		testCassandraQueueV2MultiplePartitions(t, cluster)
 	})
+	t.Run("DataCorruption", func(t *testing.T) {
```

Comment:
I wanted to run QueryErrors test before DataCorruption tests. DataCorruption tests creates some corrupt entries in queue metadata table which doesn't allow me to run some QueryErrors tests for ListQueue() tests.

Reply:
I was thinking that we can only use two queueTypes normal and DLQ. I was already using that to isolate some listqueue tests. Just realized that we can use any integer for queue type here. updated the test to follow your first suggestion.